lib/tasks/db.rake

```diff
@@ -0,0 +1,21 @@
+namespace :db do
+  desc 'Erase all tables'
+  task clear: :environment do
+    conn = ActiveRecord::Base.connection
+    tables = conn.tables
```

Comment:
I would be wary deleting the tables like this. This seems to return the tables created by our application plus some internal rails/activerecord ones (e.g `schema_migrations`, `ar_internal_metadata`). If the internal ones are not correctly re-created, the follow up tasks might end up causing unexpected behaviours.

As long as we're comfortable on this, 👍